src/chrome/content/rules/Turner.com.xml

```diff
@@ -1,64 +1,190 @@
+		p6.sdc.mail.turner.com
+		p7.sdc.mail.turner.com
+		p8.sdc.mail.turner.com
+		p9.sdc.mail.turner.com
+		p10.sdc.mail.turner.com
+		p11.sdc.mail.turner.com
+		p12.sdc.mail.turner.com
+		web2.mail.turner.com
+		i1.web2.mail.turner.com
+		i2.web2.mail.turner.com
+		i3.web2.mail.turner.com
+		p1.web2.mail.turner.com
+		p2.web2.mail.turner.com
+		p3.web2.mail.turner.com
+		p4.web2.mail.turner.com
+		p9.web2.mail.turner.com
+		p10.web2.mail.turner.com
+		p11.web2.mail.turner.com
+		p13.web2.mail.turner.com
+		p12.web2.mail.turner.com
+		p14.web2.mail.turner.com
+		p15.web2.mail.turner.com
+		p16.web2.mail.turner.com
+		wf.mail.turner.com
+		mercury.turner.com
+		newmail.turner.com
+		nycmail1.turner.com
+		nycmail2.turner.com
+		nycnn-dnga.turner.com
+		nycowa01.turner.com
+		ppmdvmsa.turner.com
+		search.turner.com
+		smtpvip.turner.com
+		support.turner.com
+		alt.tcm.turner.com
+		tennews.turner.com
+		s1.tennews.turner.com
+		s2.tennews.turner.com
+		s3.tennews.turner.com
+		s4.tennews.turner.com
+		tunnel.turner.com
+		vc.turner.com
+
+	Unreachable:
+		autodiscover.turner.com
+		lonmail1.turner.com
 -->
-<ruleset name="Turner.com (partial)" default_off="Needs ruleset tests">
-
-	<target host="*.turner.com" />
-		<!--
-			Attempt at fixing CNN video streaming:
-								-->
-		<exclusion pattern="^http://(?:i\d?|s|z)\.cdn\.turner\.com/.*(?:crossdomain\.xml|\.swf)(?:$|\?)" />
-
-			<!--	-ve:
-					-->
-			<test url="http://i.cdn.turner.com/adultswim/big/img/global/adult-swim-logo.png" />
-			<test url="http://i.cdn.turner.com/adultswim/tools/css/global.css" />
-			<test url="http://s.cdn.turner.com/adultswim/footer/legal/tools/img/ad-choices-logo-small.gif" />
-
-
-	<rule from="^http://(?:i\d?|s|z)\.cdn\.turner\.com/"
-		to="https://s.cdn.turner.com/" />
-
+<ruleset name="turner.com">
+	<target host="turner.com" />
+	<target host="www.turner.com" />
+	<target host="asiapacific.turner.com" />
+	<target host="ttv-content-asperatx.awsbest.turner.com" />
+	<target host="ttv-content-asperatx-all.awsbest.turner.com" />
+	<target host="ttv-content-dev-asperatx.dev.awsbest.turner.com" />
+	<target host="ttv-content-dev-asperatx-all.dev.awsbest.turner.com" />
+	<target host="cvp1.cdn.turner.com" />
+	<target host="ht.cdn.turner.com" />
+	<target host="i.cdn.turner.com" />
+	  <test url="http://i.cdn.turner.com/trutv/trutv.com/graphics/photos/serial_killers/notorious/zodiac/David-Arthur-Faraday200.jpg" />
+	<target host="s.cdn.turner.com" />
+	<target host="z.cdn.turner.com" />
+	<target host="cnnislink.turner.com" />
+	<target host="zeus.cp-ext.turner.com" />
+	<target host="development.turner.com" />
+	<target host="dngupload.turner.com" />
+	<target host="emea.turner.com" />
+	<target host="help.turner.com" />
+	<target host="international.turner.com" />
+	<target host="pm-reporter.iso.turner.com" />
+	<target host="mozart.turner.com" />
+	<target host="music.turner.com" />
+	<target host="news.turner.com" />
+	<target host="prensa.turner.com" />
```

Comment:
`prensa.turner.com` appeared multiple times in this ruleset, please remove the duplicates within the same file. thanks.